src/components/Footer.scss

```diff
@@ -1,9 +1,9 @@
-.Footer {
-  height: 1rem;
-  composes: Montserrat from '../styles/fonts.scss';
```

Comment:
This is an old version of the file. Are you sure you reviewed the latest version?